_includes/social-icons/reddit.svg

```diff
@@ -0,0 +1,2 @@
+
+<svg xmlns="http://www.w3.org/2000/svg" width="100%" viewBox="0 0 167.6 167.6"><circle class="social-icon-bg" cx="83.8" cy="83.8" r="83.8"/><path class="social-icon-inside" d="M139.7,83.8a12.18,12.18,0,0,0-12.2-12.2A12,12,0,0,0,119,75c-8.4-6-19.9-9.9-32.7-10.4l5.6-26.2,18.2,3.9a8.71,8.71,0,1,0,8.4-9,8.88,8.88,0,0,0-7.5,4.8L90.7,33.8a2.13,2.13,0,0,0-1.6.3,1.94,1.94,0,0,0-.9,1.4L82,64.7C69,65.1,57.3,69,48.8,75.1a12.23,12.23,0,0,0-17,17.6A12.75,12.75,0,0,0,35.2,95a22.18,22.18,0,0,0-.3,3.7c0,18.8,21.9,34.1,48.9,34.1s48.9-15.2,48.9-34.1a23.08,23.08,0,0,0-.3-3.7,12.24,12.24,0,0,0,7.3-11.2ZM55.9,92.5a8.7,8.7,0,0,1,17.4,0h0a8.7,8.7,0,0,1-17.4,0Zm48.7,23.1c-6,6-17.4,6.4-20.7,6.4s-14.8-.5-20.7-6.4a2.26,2.26,0,0,1,3.2-3.2c3.8,3.8,11.8,5.1,17.5,5.1s13.8-1.3,17.5-5.1a2.23,2.23,0,0,1,3.2,0A2.43,2.43,0,0,1,104.6,115.6ZM103,101.3a8.7,8.7,0,0,1,0-17.4h0a8.7,8.7,0,1,1,0,17.4Z"/></svg>
```

Comment:
Seems like some of the icons use a secondary color while others use transparency. To make it look more professional we should stick to 1 option for all of them as that will look more consistent. I personally like the transparent look better so let's go with that option for all if you don't mind.